appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -45,3 +45,7 @@ title Violations
 
 title "Passed?"
 ./ec-pass-fail.sh "${output}" | tee "${passed}"
+
+if [[ "${STRICT_POLICY:-'1'}" = "1" ]]; then
+    [[ "$(cat ${passed})" = "true" ]]
+fi
```

Comment:
1. Not sure how the if it's needed, but maybe this for extra clarity:

```
     else
         true
     fi
```
Update: set -e means it just immediately fails, right so never mind about this..

2. Maybe it's cleaner to move the exit code into ec-pass-fail.sh. You could pass in the param perhaps:
    
```
     ./ec-pass-fail.sh "${output}" "${STRICT_MODE:-1}" | tee "${passed}"
```

But actually the -o pipefail might mess that up so then maybe you'd have to pass the output file in as well, idk.

Reply:
This is just an idea to consider, not a blocker for merging.

Reply:
It's much nicer now with the comment and the usage of `==`. I'm leaving it as is but am open to suggestions.